docs/manual/creating-a-checker.tex

```diff
@@ -2480,9 +2480,13 @@
 You do not have to understand the implementation of javac to
 build powerful and useful checkers.
 Beyond this document,
-other useful resources include the Java Infrastructure
-Developer's guide at
-\url{https://netbeans.apache.org/wiki/Java_DevelopersGuide.asciidoc} and the compiler
+%% TODO: this link no longer works.  Is there a better one?
```

Comment:
I often change a broken link to a Wayback Machine link if the content is valuable.  Either that or commenting out is fine.